sys/drv/rtl8139.c

```diff
@@ -32,18 +37,72 @@ static int rtl_reset(rtl8139_state_t *state) {
   return 1;
 }
 
-static int rtl8139_attach(device_t *dev) {
+static intr_filter_t rtl8139_intr(void *data) {
+  rtl8139_state_t *state = data;
+  // TODO: add enqueue routines
+  bus_write_2(state->regs, RL_ISR, RL_ISR_RX_OK);
+  return IF_FILTERED;
+}
```

Comment:
Do I understand correctly that for each interrupt you **only** write `RL_ISR_RX_OK`?
Shouldn't we be doing something more with this data?

Reply:
you are right, all expected routines are implemented in #900 - I wanted to separate this PR into 2 PRs to be more readable (one for setting-up interrupts, 2nd for the implementation of rx-path)